src/components/calcite-shell-center-row/calcite-shell-center-row.e2e.ts

```diff
@@ -0,0 +1,66 @@
+import { newE2EPage } from "@stencil/core/testing";
+
+import { CSS } from "./resources";
+import { accessible, defaults, hidden, renders } from "../../tests/commonTests";
+
+describe("calcite-shell-center-row", () => {
+  it("renders", async () => renders("calcite-shell-center-row"));
+
+  it("honors hidden attribute", async () => hidden("calcite-shell-panel"));
```

Comment:
`hidden("calcite-shell-panel"))` should be `calcite-shell-center-row`

Reply:
I did dat now.